modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360EnricherImpl.java

```diff
@@ -101,17 +102,29 @@ private void mapExternalIdsOnSW360Release(SW360Release sw360Release, Artifact ar
     }
 
     private void addLicenseFact(Optional<String> licenseRawData, Artifact artifact, Function<LicenseInformation, ArtifactLicenseInformation> licenseCreator, boolean isAlreadyPresent) {
-        Optional<ArtifactLicenseInformation> licenseExpression = licenseRawData
-                .map(LicenseSupport::fromSPDXExpression)
-                .map(licenseCreator);
+        licenseRawData.map(this::parseSpdxExpression)
+                .map(licenseCreator)
+                .map(expression -> logOverwriteLicenseInformation(artifact, isAlreadyPresent, expression))
+                .ifPresent(artifact::addFact);
+    }
 
-        if (isAlreadyPresent && licenseExpression.isPresent()) {
+    private ArtifactLicenseInformation logOverwriteLicenseInformation(Artifact artifact, boolean isAlreadyPresent, ArtifactLicenseInformation expression) {
+        if (isAlreadyPresent) {
             warnAndReport(artifact,
-                    "License information of type " + licenseExpression.get().getClass().getSimpleName() + " found in SW360. Overwriting existing information in artifact.",
+                    "License information of type " + expression.getClass().getSimpleName() + " found in SW360. Overwriting existing information in artifact.",
                     MessageType.OVERRIDE_ARTIFACT_ATTRIBUTES);
         }
+        return expression;
+    }
 
-        licenseExpression.ifPresent(artifact::addFact);
+    private LicenseInformation parseSpdxExpression(String expression) {
+        try {
+            return LicenseSupport.fromSPDXExpression(expression);
+        } catch (SpdxException e) {
+            License backup = new License();
```

Comment:
Nit: Maybe better "fallback" instead of "backup"? Or yet better something more telling, like "licenseFromExpression" or "unparsed(License)Expression"?

Reply:
Used unparseableExpression as name.